roles/kubernetes/preinstall/tasks/set_facts.yml

```diff
@@ -25,7 +25,7 @@
 - set_fact: etcd_access_address="{{ access_ip | default(etcd_address) }}"
 - set_fact: etcd_peer_url="http://{{ etcd_access_address }}:2380"
 - set_fact: etcd_client_url="http://{{ etcd_access_address }}:2379"
-- set_fact: etcd_authority="{{ access_ip|default('127.0.0.1') }}:2379"
```

Comment:
this should be tested with cloud provider based deployments and access_ip is not null